infra/testApi.mjs

```diff
@@ -2,6 +2,7 @@ import { strict as assert } from 'node:assert';  // use strict equality everywhe
 
 // Future TODO: before this API becomes set in stone/offered publicly, we should change the results of these methods to be just the output data rather than duplicating input values.
 
+
```

Comment:
It is a good habit generally to review your diff before you request a review, and if you have useless whitespace changes like this to get rid of them.